halyard-core/src/main/java/com/netflix/spinnaker/halyard/core/resource/v1/ObjectResource.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 Armory, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.halyard.core.resource.v1;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+
+public class ObjectResource extends ObjectReplaceTemplatedResource {
```

Comment:
This extends the `ObjectReplaceTemplatedResource` class above, not entirely sure this class is necessary as contents can be part of the above class, would just have to make it not abstract. I matched the implementation of the StringReplaceTemplatedResource